.connector-store/meta.json

```diff
@@ -6,11 +6,13 @@
     "description": "Amazon SQS offers reliable and scalable hosted queues for storing messages as they travel between computers. By using Amazon SQS, you can move data between distributed components of your applications that perform different tasks without losing messages or requiring each component to be always available. The Amazon SQS connector for WSO2 EI uses the Amazon SQS API and allows you to send and receive messages, facilitating asynchronous messaging between integrated systems.",
     "status": "Active",
     "labels": [
-        "queues"
+        "queues",
+        "AWS sqs",
+        "SQS"
     ],
     "releases": [
         {
-            "tagName": "v1.1.6",
+            "tagName": "v2.0.0",
```

Comment:
We should add a new entry here instead of renaming. Users who used 1.1.6 won't be able to download that connector from the store after this change.

We may need to have 1.1.6 as well, maybe marked as deprecated. @arunans23 @rosensilva WDYT?

Reply:
Added in https://github.com/wso2-extensions/esb-connector-amazonsqs/pull/45